server/controllers/reports_proposed/data/debitor_group_report.js

```diff
@@ -0,0 +1,77 @@
+// reports_proposed/data/balance_sheet.js
+// Collects and aggregates data for the enterprise balance sheet
+var q       = require('q');
+var db      = require('../../../lib/db');
+var numeral = require('numeral');
+var sanitize = require('../../../lib/sanitize');
```

Comment:
We are trying to remove sanitize.js.  (See #329).  Could you please remove all code that references it?